### PR TITLE
Add a puppeteer-based benchmark

### PR DESCRIPTION
**Pull request by davidmanzanares**

### CODE REVIEW

**Comment by IagoLast** [comment]
Context?

**Comment by davidmanzanares** [comment]
Easier way to benchmark the renderer by automatically launch one example and track some metrics like average time, P99...

Not super complete, but better than nothing and really simple.

Usage: launch the script and wait for the metrics on the terminal.

File: profile.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const puppeteer = require('puppeteer');
+
+
+puppeteer.launch({
+    headless: false,
+    args: ['--disable-gpu-vsync'],
+}).then(browser => {
+    browser.newPage().then(page => {
+        page.goto('file:///home/dmanzanares/github/renderer-prototype/examples/expressions/animation.html', { waitUntil: 'networkidle0' }).then(() => {
```
Comment (Jesus89) [inline profile.js]: Use __dirname here

**Comment by Jesus89** [comment]
This should be moved to `test/benchmark` directory and use its own `html` example for the test. It should be nice to have also the command `yarn test:benchmark` or similar.

File: profile.js (hunk 1 above)
Comment (davidmanzanares) [reply profile.js]: ups

Comment (IagoLast) [reply profile.js]:
```js
 const puppeteer = require('puppeteer');
+const path = require('path');
 
+const TEST_FILE = path.join(__dirname, '/examples/expressions/animation.html');
 
 puppeteer.launch({
     headless: false,
     args: ['--disable-gpu-vsync'],
 }).then(browser => {
     browser.newPage().then(page => {
-        page.goto('file:///home/dmanzanares/github/renderer-prototype/examples/expressions/animation.html', { waitUntil: 'networkidle0' }).then(() => {
+        page.goto(`file://${TEST_FILE}`, { waitUntil: 'networkidle0' }).then(() => {
```

**Comment by davidmanzanares** [comment]
:+1: @Jesus89 can you re CR it?

**Pull request merged**